.flake8

```diff
@@ -0,0 +1,6 @@
+[flake8]
+max-line-length = 127
+max-complexity = 10
+count = true
+statistics = true
+exclude = .venv
```

Comment:
Locally I use "venv" instead of ".venv" - does the linter give errors if you remove this line? I believe the linting looks at the gitignore as well but not sure since both venv and .venv are gitignored

Plus the command that is being run is always "flake8 atomic_agents" which limits it to the atomic_agents directory

So, I believe the exclude can go as well

Reply:
Done